### PR TITLE
fix: broken MNIST download in e2e tests

### PR DESCRIPTION
**Pull request by rb-determined-ai**

### CODE REVIEW

File: e2e_tests/tests/cluster/test_slurm.py [hunk 1]
```diff
@@ -191,6 +191,7 @@ def test_master_host() -> None:
 @pytest.mark.skipif(not torch.cuda.is_available(), reason="no gpu available")
 @pytest.mark.e2e_slurm
 @pytest.mark.e2e_pbs
+@pytest.mark.rbdebug  # XXX
```
Comment (dannysauer) [inline e2e_tests/tests/cluster/test_slurm.py:194]: Drunken comment?

**Review by dannysauer — APPROVED** [review]
LGTM

**Review by gt2345 — APPROVED** [review]
LGTM

File: e2e_tests/tests/cluster/test_slurm.py (hunk 1 above)
Comment (rb-determined-ai) [reply e2e_tests/tests/cluster/test_slurm.py:194]: I am `rb`, `debug` is what I'm doing, and XXX is my tag to not forget to revisit a line before landing it.

**Review by corban-beaird — APPROVED** [review]
Looks great!

**Pull request merged**